scripts/k8s/launch_enclave.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Enclave Node
+bash -c '\
+    export CLUSTER_NAME=testenclave1 CLUSTER_HTTP_PORT=9083 DEVSPACE_PROFILE=enclave && \
+    tox -e dev.k8s.start -- --volume /sys/kernel/security:/sys/kernel/security --volume /dev/tmprm0:/dev/tmprm0 && \
+    tox -e dev.k8s.hotreload'
```

Comment:
i'm thinking since you're creating these scripts - might as well make them tox commands?

`tox -e dev.k8s.launch.domain`
`tox -e dev.k8s.launch.gateway`
`tox -e dev.k8s.launch.enclave`

Reply:
That's a good idea , adding them now 👍

Reply:
Resolved in [2c15bd0](https://github.com/OpenMined/PySyft/pull/8610/commits/2c15bd0f152c41855c63c4a1da20fa5b1f124597)

Reply:
I have added  a posargs to tox, so we could easily switch between deploy and hotreload mode.

Domain in Deploy Mode

```sh
tox -e dev.k8s.launch.domain
```
Domain in Hot Reload Mode

```
tox -e dev.k8s.launch.domain -- hotreload
```